thunder/tests/test_jit_general.py

```diff
@@ -639,6 +640,7 @@ def test_litgpt_variants(name, device):
 
     x = torch.randint(0, 200, (5, 5), device=device)
     config = litgpt_model.Config.from_name(name)
+    config.n_layer = 1
```

Comment:
I am just curious. Was this value set to `1` before?

Reply:
No, I think it's better to set it to 1 to make the test run faster, there's no need to run the full for-loop of the transformer. I don't want to create another "-like" variant as done for other configs https://github.com/Lightning-AI/lightning-thunder/blob/4d9fa6057dbff420491861e82985f0cfc3dcc31e/thunder/tests/litgpt_model.py#L8-L116

Reply:
Given that we might use them in various places, creating a like might be worthwhile.
We're usually using 2 layers because it includes layer->layer data flow.

Reply:
I'll revert this change but will keep the requirements bump.